### ISSUE TITLE
Changing name of .acs

### ISSUE THREAD

**Issue by gcupani** [DAS-OATs/astrocook#351]
When the name of an `.acs` is changed from the filesystem, it cannot be opened any more.

**Comment by G-Francio** [comment]
This was fixed at some point, and the fix seems to have disappeared. Might be fixed as part of the changes in #334

**Comment by gcupani** [comment]
You are right, I will check that

**Comment by gcupani** [comment]
Apparently it isn't the same issue. The changes that solved the problem in the past (issue #223, 192a852) are propagated into the current version of `develop`, yet for some specific `.acs` an error is displayed:

```
  File "/Users/guido/GDrive/GitHub/astrocook/ac_gui.py", line 17, in main
    gui = GUI(paths, flags)
  File "/Volumes/GoogleDrive-110625037481089626894/My Drive/GitHub/astrocook/astrocook/gui.py", line 82, in __init__
    self._panel_sess._on_open(os.path.realpath(p))
  File "/Volumes/GoogleDrive-110625037481089626894/My Drive/GitHub/astrocook/astrocook/gui.py", line 542, in _on_open
    self._gui._panel_sess._on_add(sess, open=True)
  File "/Volumes/GoogleDrive-110625037481089626894/My Drive/GitHub/astrocook/astrocook/gui.py", line 501, in _on_add
    ko = self._gui._sess_sel.open()
  File "/Volumes/GoogleDrive-110625037481089626894/My Drive/GitHub/astrocook/astrocook/session.py", line 360, in open
    self._data_iden(hdul, hdr)
```

**Comment by gcupani** [comment]
The problem arises when the `.acs` archive has been renamed and is unpacked within a directory that already contains a `*_spec.fits` spectrum.
